spec/controllers/state_file/questions/nj_tenant_eligibility_controller_spec.rb

```diff
@@ -44,7 +44,7 @@
     end
 
     context "when a user is not MFS" do
-      let(:intake) { create :state_file_nj_intake, :df_data_mfj }
+      let(:intake) { create :state_file_nj_intake, :married_filing_jointly }
```

Comment:
[dust] why this change?

Reply:
I got rid of `df_data_mfj` altogether. To instead specifically have the DF traits indicate if spouse/primary is a claimed dependent. All these changes are referencing DF data when really all they care about is the filing status itself. No need to over-do the setup